nncf/quantization/algorithms/min_max/algorithm.py

```diff
@@ -540,6 +540,7 @@ def _get_quantization_target_points(
             self._backend_entity.shapeof_metatypes,
             self._backend_entity.dropout_metatypes,
             self._backend_entity.read_variable_metatypes,
+            nncf_graph_contains_constants=backend != BackendType.TORCH,
```

Comment:
I'd like to note that we introduce backend-specific logic into a common part of the PTQ, which does not fit with our approach. Do you agree with that @alexsu52, @AlexanderDokuchaev, @nikita-savelyevv?

Reply:
Yes, I agree. @daniil-lyakhov Could you update `filter_constant_nodes` pass for discontinuous graphs?

Reply:
Done, please check

Reply:
I tried implementation we discussed db0ad8df19da7e4c796e17dbc876c4ea9c9f2e36, but found a corner case that is breaking the pass:
Before:
![image](https://github.com/openvinotoolkit/nncf/assets/74656388/215e620f-5594-4ceb-975d-f3ad9bbb4632)
Expected after:
![image](https://github.com/openvinotoolkit/nncf/assets/74656388/66358889-f922-4657-8355-495ccf9500bf)
Actual after:
![image](https://github.com/openvinotoolkit/nncf/assets/74656388/9f6a50f7-f8a6-431b-9c14-9a906602bf86)
That's because convolution node should be an input node so survive filtering, but we have no prior information yet that can help us to mark this convolution as an input node